src/cephadm/cephadm

```diff
@@ -96,45 +96,36 @@ class Monitoring(object):
 
     components = {
         "prometheus": {
-            "image": {
-                "image": "prom/prometheus:latest",
-                "cpus": '2',
-                "memory": '4GB',
-                "args": [
-                    "--config.file=/etc/prometheus/prometheus.yml",
-                    "--storage.tsdb.path=/prometheus",
-                    "--web.listen-address=:{}".format(port_map['prometheus']),
-                ]
-            },
+            "cpus": '2',
+            "memory": '4GB',
+            "args": [
+                "--config.file=/etc/prometheus/prometheus.yml",
+                "--storage.tsdb.path=/prometheus",
+                "--web.listen-address=:{}".format(port_map['prometheus']),
+            ],
             "config-json": [
                 "prometheus.yml",
             ],
         },
         "node-exporter": {
-            "image": {
-                "image": "prom/node-exporter",
-                "cpus": "1",
-                "memory": "1GB",
-                "args": [
-                    "--no-collector.timex",
-                ],
-            }
+            "cpus": "1",
+            "memory": "1GB",
+            "args": [
+                "--no-collector.timex",
+            ],
         },
         "grafana": {
-            "image": {
-                "image": "pcuzner/ceph-grafana-el8:latest",
```

Comment:
Hm. removing this means, we no longer have a **working** default for the grafana container?

Reply:
Re-added the default image key, but it would be good to provide a `ceph-grafana` image built from the upstream repo .. 

This PR can be rebased once https://github.com/ceph/ceph/pull/32580 is merged.